### ISSUE TITLE
ModalPane does not interact properly with screenreaders

### ISSUE THREAD

**Issue by crschnick** [mkpaz/atlantafx#58]
When using the Windows screen reader, the contents of a ModalPane are still recognized even though they should not be visible anymore after calling hide().

**Comment by mkpaz** [comment]
I suppose it’s another JavaFX bug and the screen reader doesn’t take the view order property into account. I’ve checked the list of AccessibleActions and failed to see something suitable.

**Comment by crschnick** [comment]
Can't you just make it invisible/not managed/not focus traversable when the pane is hidden?

**Comment by mkpaz** [comment]
Hard to say as it could break the content animations. I would prefer to avoid this level of complexity.

**Comment by mkpaz** [comment]
Closing in favor of https://github.com/mkpaz/atlantafx/issues/63 as it's more generic issue.
